pkg/kubectl/resource_printer.go

```diff
@@ -1021,32 +957,37 @@ func printIngress(ingress *extensions.Ingress, w io.Writer, options PrintOptions
 		return err
 	}
 
-	// Lay out all the rules on separate lines.
-	extraLinePrefix := ""
-	if options.WithNamespace {
-		extraLinePrefix = "\t"
-	}
-	for _, rules := range hostRules {
-		if rules.HTTP == nil {
-			continue
-		}
-		_, err := fmt.Fprintf(w, "%s\t%v\t", extraLinePrefix, rules.Host)
-		if err != nil {
-			return err
-		}
-		if _, err := fmt.Fprint(w, appendLabelTabs(options.ColumnLabels)); err != nil {
-			return err
+	// Lay out all the rules on separate lines if use wide output.
```

Comment:
I begrudgingly concede, but one almost always wants to see the host and urls associated with an IP for Ingress. Even if output is getting piped through awk/grep, you're probably looking for more than just the ip.
 
That said, printing just the IP makes implementing kubectl get a lot easier: https://github.com/kubernetes/kubernetes/issues/22837